config/routes.rb

```diff
@@ -3,10 +3,12 @@
     resources :needs, only: [:new, :create]
   end
   get '/contacts/:id/needs', to: 'contacts#show_needs'
+  resources :contact_profiles, only: [:edit, :update]
```

Comment:
Hm, not sure about the name for this... Probably makes more sense as a nested route under the `/contacts` resource (although I know it maps to a different controller), like `/contacts/123/triage` or something. Maybe we call it the triage controller while we're at it?

Reply:
Yep that's right, it didn't seem to make sense for it to be mixed with the needs_controller functionality any more.  I struggled for what to name it though.  Triage sounds good though, will change it to that.